sympy/stats/error_prop.py

```diff
@@ -0,0 +1,74 @@
+"""Tools for arithmetic error propogation."""
+from __future__ import print_function, division
+from itertools import repeat
+
+from sympy import Symbol, symbols, Add, Mul, simplify, Pow, exp
```

Comment:
I wonder if there is something similar for `vim`. @asmeurer how did you find out that `symbols` is unused? Just by looking at it?